manifests/service.pp

```diff
@@ -1,7 +1,13 @@
 # Configure service
 #
 class fluentd::service inherits fluentd {
-
+  file {
+    '/opt/td-agent':
+      owner   => 'td-agent',
+      group   => 'td-agent',
```

Comment:
May be as an improvement we can use placeholders from whats set in the params for owner and group similar to https://github.com/acquia/puppet-fluentd/blob/master/manifests/user.pp
https://github.com/acquia/puppet-fluentd/blob/master/manifests/params.pp

```
      owner   => $::fluentd::user_name
      group   => $::fluentd::user_group,
```

Reply:
Done